macros/persist_pass_through_columns.sql

```diff
@@ -0,0 +1,9 @@
+{% macro persist_pass_through_columns(pass_through_variable) %}
+
+{% if var(pass_through_variable) %}
```

Comment:
should we add a default here in case the variable flow get's lost? My thought is below:
```suggestion
{% if var(pass_through_variable, none) %}
```
This way there is no chance dbt could throw the "variable does not exist" error.

Reply:
Thoughts?